stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -87,12 +87,15 @@ protected static void mainInternal(String[] args, Benchmark benchmark) {
    * @return the JobConfig
    * */
   public PlanConfig generateJobConfig(String[] args) {
-    // remove the cluster flag
+    // remove the cluster flag and java opts
     List<String> commandArgs =
-            Arrays.stream(args).filter((s) -> !BaseParameters.CLUSTER_FLAG.equals(s))
-                    .filter((s) -> !s.isEmpty()).collect(Collectors.toList());
+            Arrays.stream(args).filter((s) -> !BaseParameters.JAVA_OPT_FLAG.equals(s)
+                && !BaseParameters.CLUSTER_FLAG.equals(s)
+                && !s.isEmpty()
+                && !mBaseParameters.mJavaOpts.contains(s)).collect(Collectors.toList());
 
-    commandArgs.addAll(mBaseParameters.mJavaOpts);
```

Comment:
the JavaOpts is applied to the job launch process here. are you saying there is another level of launching jobs within the job?

Reply:
basically, all job tasks on the workers also spawn processes, to make it look exactly like it would have if it was run locally (not in cluster mode). https://github.com/Alluxio/alluxio/blob/master/job/server/src/main/java/alluxio/job/plan/stress/StressBenchDefinition.java#L114

I just want to make sure if we set java opts for max mem, that it will affect local mode, as well as cluster mode, all in the same way.